pywt/_multilevel.py

```diff
@@ -310,6 +312,12 @@ def waverec2(coeffs, wavelet, mode='symmetric', axes=(-2, -1)):
     a = np.asarray(a)
 
     for d in ds:
+        if not isinstance(d, tuple) or len(d) != 3:
```

Comment:
maybe `isinstance(d, (tuple, list))`? when manually constructing input, could easily be that users now using list of lists rather than list of tuples, and that currently works fine.

Reply:
done. will merge once CI passes. thanks for the review